package.json

```diff
@@ -1,13 +1,13 @@
 {
   "name": "algoliasearch-extensions-bundle",
-  "version": "4.3.9",
+  "version": "4.4.0-v3",
   "description": "Bundle autocomplete.js and instantsearch.js for use in algoliasearch extensions integrations",
   "main": "index.js",
   "scripts": {
     "watch": "webpack --watch -d & http-server dist/ & wait",
     "build-front": "VERSION=`cat package.json | json version` ./scripts/build.sh",
     "build-back": "VERSION=`cat package.json | json version` ./scripts/build-admin.sh",
```

Comment:
if you're no longer using build-back, maybe remove everything related to it too?